packages/ui/src/TapToEdit.tsx

```diff
@@ -194,29 +213,15 @@ export const TapToEdit = ({
         width="100%"
         {...rowBoxProps}
       >
-        <Box>
+        <Box flex="grow">
           <Text weight="bold">{title}:</Text>
-          {fieldProps?.type === "address" && (
-            <Box
-              onClick={
-                () =>
-                  Linking.openURL(
-                    `https://www.google.com/maps/search/?api=1&query=${encodeURIComponent(
-                      formatAddress(value, true)
-                    )}`
-                  )
-                // eslint-disable-next-line react/jsx-curly-newline
-              }
-            >
-              <Text color="blue" underline={fieldProps?.type === "address"}>
-                Google Maps
-              </Text>
-            </Box>
-          )}
+          {description && <Text>{description}</Text>}
```

Comment:
Instead of always showing the description if available, what if we have a flag we can set as a prop to show it as a tooltip instead? Just thinking showing the description all the time on some of our UIs that use this component will make it very busy.

Reply:
Ooooo I love that!